resources/views/email/auth/register.blade.php

```diff
@@ -1,7 +1,11 @@
-Thanks for signing up!
+{{ trans('auth.register.email.body.intro') }}!
 
-We just need you to confirm your email address:
-{{ route('register.confirm', [$user->registration_token]) }}
+{!! trans('auth.register.email.body.text', [
+    'link' => link_to_route('register.confirm', null, [$user->registration_token], [])
+]) !!}
 
---
-{{ config('site.name')}}
+
+</br></br>
```

Comment:
Do we need to introduce extra HTML at this point? Lets keep the mails as clean as possible.